apps/android/app/src/main/java/chat/simplex/app/views/chat/group/AddGroupMembersView.kt

```diff
@@ -237,12 +242,14 @@ fun ContactCheckRow(
     iconColor = HighOrLowlight
   } else if (checked) {
     icon = Icons.Filled.CheckCircle
-    iconColor = MaterialTheme.colors.primary
+    iconColor = if (enabled) MaterialTheme.colors.primary else HighOrLowlight
   } else {
     icon = Icons.Outlined.Circle
     iconColor = HighOrLowlight
   }
   SectionItemView(click = {
+    if (!enabled) return@SectionItemView
```

Comment:
@avently can we do `break` here?
```suggestion
    if (!enabled) break
```

Reply:
Nope. What are you trying to `break` here? This keyword works in loops (for example, in `while` loop to stop the loop).
Here `return` just skips all code block from executing